### PR TITLE
98971 style render delay

### PR DESCRIPTION
**Pull request by numbap**
## [ADO-98971](https://dev.azure.com/VP-BD/DECD/_workitems/edit/98971)

Fixed [AB#98971](https://dev.azure.com/VP-BD/d5ca3cd4-19cb-4c8c-b3e6-a7068f4677e4/_workitems/edit/98971)

### Description

List of proposed changes:

- Updated fonts so that the load using block instead of swap, in order to get rid of CSS "jiggle" during page load.
-

### CODE REVIEW

**Review by nibivi77 — APPROVED** [review]
Seems to be working fine! Good find Pat!

File: next-env.d.ts [hunk 1]
```diff
@@ -0,0 +1,5 @@
+/// <reference types="next" />
+/// <reference types="next/image-types/global" />
+
```
Comment (shewood) [inline next-env.d.ts:3]: What is this file for?

File: pages/contact-us/contact-employment-insurance.js [hunk 2]
```diff
@@ -27,7 +27,10 @@ export default function ContactEmploymentInsurance(props) {
       data-testid="contactEI-test"
       data-cy="eIContactUsContent"
     >
-      <Heading id="my-dashboard-heading" title={props.pageContent.title} />
+      <Heading
+        id="my-dashboard-heading"
+        title={`${props.pageContent.title} XXX`}
```
Comment (shewood) [inline pages/contact-us/contact-employment-insurance.js:32]: what does the XXX do?

Comment (numbap) [reply pages/contact-us/contact-employment-insurance.js:32]: I'm still testing in Teamcity. Not ready to merge yet.

File: next-env.d.ts (hunk 1 above)
Comment (numbap) [reply next-env.d.ts:3]: Possibly a typo.

**Review by shewood — CHANGES_REQUESTED** [review]
the env file shouldn't be added

**Comment by numbap** [comment]
Create new branch on 98971-style-render-delay_new

**Pull request closed without merging**